core/src/main/java/org/apache/carbondata/core/datastore/page/UnsafeFixLengthColumnPage.java

```diff
@@ -201,12 +201,18 @@ public void putDouble(int rowId, double value) {
 
   @Override
   public void putBytes(int rowId, byte[] bytes) {
+    try {
+      ensureMemory(eachRowSize);
+    } catch (MemoryException e) {
```

Comment:
MemoryException can be runtime exception

Reply:
ok Will handle this in different PR